action.yml

```diff
@@ -23,15 +23,18 @@ inputs:
   checkUser:
     description: 'Enforce organization membership for GitHub user'
     required: false
-    default: false
+    default: 'false'
```

Comment:
Should these be defined as Boolean?
Does that impact if we need to use a string false vs an actual boolean false? 

https://github.com/actions/toolkit/pull/725/files
https://yaml.org/type/bool.html

Reply:
Yeah, this one is strange because I originally had the value set as an actual boolean `false`, but remember getting some kind of runtime error when running the action locally. Let me change it back now and see if that still happens.